conda-recipes/tokenizers-feedstock/recipe/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "tokenizers" %} 
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/huggingface/tokenizers/archive/python-v{{ version }}.tar.gz
+  sha256: fc27e2f0e39dacf1d14442482874ba2da2f5a48a969bdf8cd1eeffe0e9d46099
+
+build:
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+    - rust-nightly
+    - setuptools-rust
+  run:
+    - typing
+    - python
+
+test:
+  commands:
```

Comment:
Just curious if this would be same test:

```
test:
  imports:
  - tokenizers
  - tokenizers.CharBPETokenizer
  ...
```

Reply:
I believe so, yes.

Reply:
I tried that first, but the direct import like that doesn't work, so I switched to the `from tokenizers import ....` form. Not sure why the straight import doesn't work, but the `from ...` version is also what the `transformers` package uses, so this testing mimics the use there.

Reply:
sounds good. Thank you.